docs/supported_entities.md

```diff
@@ -1,13 +1,61 @@
-# Presidio supported PII entities
+# PII entities supported by Presidio
 
-## Description
+Presidio contains predefined recognizers for PII entities.
+This page describes the different entities Presidio can detect and the method Presidio employs to detect those.
 
-## Table of contents
+In addition, Presidio allows you to add custom entity recognizers.
+For more information, refer to the [adding new recognizers documentation](analyzer/adding_recognizers.md).
```

Comment:
same comment as @shiranr below - as this document is not there at the time of this PR